cw/cw305/capture.py

```diff
@@ -361,6 +361,33 @@ def aes_fvsr_key(ctx: typer.Context,
     capture_end(ctx.obj.cfg)
 
 
+@app_capture.command()
+def aes_mix_column(ctx: typer.Context,
+                   num_traces: int = opt_num_traces,
+                   plot_traces: int = opt_plot_traces):
+    """Capture AES traces. Fixed key, Random texts. 4 sets of traces. Mix Column HD CPA Attack.
+    Attack implemented by ChipWhisperer:
+    Repo: https://github.com/newaetech/chipwhisperer-jupyter/blob/master/experiments/MixColumn%20Attack.ipynb # noqa: E501
+    Reference: https://eprint.iacr.org/2019/343.pdf
+    See mix_columns_cpa_attack.py for attack portion.
+    """
+    capture_init(ctx, num_traces, plot_traces)
+
+    ctx.obj.ktp = cw.ktp.VarVec()
```

Comment:
why are the lines that set `key_len` and `text_len` are dropped here?

Reply:
Thanks Alphan. I've dropped them by mistake. Should I add them with another PR?

Reply:
Yes, I think that would be the best.

Reply:
Done it: https://github.com/lowRISC/ot-sca/pull/73